app/buoy_barn/urls.py

```diff
@@ -13,11 +13,18 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+from django.conf import settings
 from django.contrib import admin
 from django.urls import path, include
 
 urlpatterns = [
-    path('admin/', admin.site.urls),
-    path('api/', include('deployments.urls')),
-    path('api-auth/', include('rest_framework.urls'))
+    path("admin/", admin.site.urls),
+    path("api/", include("deployments.urls")),
+    path("api-auth/", include("rest_framework.urls")),
 ]
+
+
+if settings.DEBUG:
+    import debug_toolbar
```

Comment:
Codacy found an issue: [Unable to import 'debug_toolbar' (import-error)](https://app.codacy.com/manual/gmri/buoy_barn/pullRequest?prid=6198573)